third_party/amd/backend/compiler.py

```diff
@@ -71,8 +71,8 @@ def __post_init__(self):
         object.__setattr__(self, 'extern_libs', tuple(extern_libs.items()))
         assert self.num_warps > 0 and (self.num_warps & (self.num_warps - 1)) == 0, \
                "num_warps must be a power of 2"
-        if(self.matrix_core_version == -1):
-            object.__setattr__(self, 'matrix_core_version', self.get_matrix_core_version(self.arch))
+        if(self.mfma_version == -1):
+            object.__setattr__(self, 'mfma_version', self.get_mfma_version(self.arch))
```

Comment:
So on Navi arch, this is set to mfma_version = 0. Then the stream-pipeline pass is disabled.
Is this the expected behavior on Navi?

Reply:
I just checked. Stream pipeline do not support WMMA for now ... So it is Ok to be disabled.
Should I add TODO comment or create an issue?

Reply:
Good point. Thanks.
Just checked again. Stream pipeline works fine.
Please, check updated https://github.com/openai/triton/pull/3309/files#diff-33c9a103282c05c9d9d213b94450ae7481b6db8c3c6d810f54f175b4735a3c72R39-R44